packages/sdk-core/previous-versions-testing/queryTests.ts

```diff
@@ -1,5 +1,20 @@
 import { Query } from "@superfluid-finance/sdk-core/src";
 import { expect } from "chai";
+import {
+    ETH_GOERLI_CHAIN_ID,
+    MATIC_CHAIN_ID,
```

Comment:
These imports look ugly cosmetically. :smile: Would have preferred an enum or something.

Reply:
true, can be something to refactor for SDK-core in the future, but there were some strange type issues w/ enums in the past that were kinda annoying